src/vm/fastserializer.cpp

```diff
@@ -12,7 +12,7 @@
 // As a result of work on V3 of Event Pipe (https://github.com/Microsoft/perfview/pull/532) it got removed
 // if you need it, please use git to restore it
 
-IpcStreamWriter::IpcStreamWriter(IpcStream *pStream) : _pStream(pStream)
```

Comment:
Should we have a separate file, other than FastSerializer.cpp, to implement the IpcStreamWriter.
It feels wrong.

Reply:
We could.

Reply:
I agree here with you, but unless anybody objects, I will rather not fix it in this PR.